### PR TITLE
Matches priority

### PR DESCRIPTION
**Pull request by audebert**

### CODE REVIEW

File: python-lib/prologin/masternode/concoursquery.py [hunk 1]
```diff
@@ -62,6 +62,8 @@
             stechec_match.status = %(match_status)s
           GROUP BY
             stechec_match.id, stechec_map.id
+          ORDER BY
+            stechec_match.priority DESC
```
Comment (Zopieux) [inline python-lib/prologin/masternode/concoursquery.py]: I'd make the ordering (per priority) deterministic to help with debugging and reproducibility and such. Add `stechec_match.id ASC` maybe?

File: python-lib/prologin/concours/stechec/models.py [hunk 2]
```diff
@@ -327,6 +336,12 @@ class Match(ExportModelOperationsMixin('match'), models.Model):
         verbose_name="carte",
     )
```
Comment (seirl) [inline python-lib/prologin/concours/stechec/models.py:338]: (nit: unwanted blank line)

File: python-lib/prologin/masternode/concoursquery.py (hunk 1 above)
Comment (audebert) [reply python-lib/prologin/masternode/concoursquery.py]: Done.

**Comment by audebert** [comment]
> I think these priority systems are generally better implemented using some kind of probability scheduling. [...]

In the context of the contest the naive priority queue is expect to meet our needs.

One problem we may have is running 2 different tournaments in parallel. With the current implementation the second tournament would only start after the first one is completed. In this case the solution is to set the matches priority to N + random value, which simulates probabilistic scheduling with a deterministic scheduler.

**Comment by audebert** [comment]
We'll also need to expose the match priorities to the web interface, at least for organizers. I suggest adding that feature in a second PR.

**Pull request merged**